src/components/Explain.jsx

```diff
@@ -1,5 +1,4 @@
 import React, { useState } from 'react'
-import { Object } from 'es6-shim';
```

Comment:
This I think got auto-imported with my `Object.entries` at some point, but now that we have babel I think it's no longer necessary.